### ISSUE TITLE
barman-cloud-backup can't load more than 200Gb

### ISSUE THREAD

**Issue by mkdel** [2ndquadrant-it/barman#284]
Hi,
only 1k chunks available to upload to S3. Can we increase this number?

```
2020-05-14 21:14:51,668 INFO Uploading test_230Gb_notcompressed/base/20200514T203640/data.tar, part 9998 (worker 0)
2020-05-14 21:14:51,933 INFO Uploading test_230Gb_notcompressed/base/20200514T203640/data.tar, part 9999 (worker 1)
2020-05-14 21:14:52,228 INFO Uploading test_230Gb_notcompressed/base/20200514T203640/data.tar, part 10000 (worker 0)
2020-05-14 21:14:52,439 INFO Uploading test_230Gb_notcompressed/base/20200514T203640/data.tar, part 10001 (worker 1)
2020-05-14 21:14:52,502 ERROR Upload error: An error occurred (InvalidArgument) when calling the UploadPart operation: Part
number must be an integer between 1 and 10000, inclusive (worker 1)
2020-05-14 21:14:52,503 INFO Uploading test_230Gb_notcompressed/base/20200514T203640/data.tar, part 10002 (worker 1)
2020-05-14 21:14:52,564 ERROR Error received from upload worker: An error occurred (InvalidArgument) when calling the Upload
Part operation: Part number must be an integer between 1 and 10000, inclusive
2020-05-14 21:14:52,565 INFO Got abort request: upload cancelled (worker 0)
2020-05-14 21:14:52,565 INFO Skipping test_230Gb_notcompressed/base/20200514T203640/data.tar, part 10003 (worker 0)
2020-05-14 21:14:52,567 INFO Got abort request: upload cancelled (worker 1)
2020-05-14 21:14:52,568 INFO Upload process stopped (worker 1)
2020-05-14 21:14:52,569 INFO Upload process stopped (worker 0)
```

**Comment by mnencia** [comment]
barman-cloud-backup uploads using 20MB chunks. the maximum number of chunks is 10k, so it throws an error after 200GB. We could increase the chunk size, but I believe that the right solution is to split the upload after reaching a reasonable size. I would propose 100 GB for that value.

**Comment by mnencia** [comment]
closed by 8a9d8e60cff6224fee0459b0cd43d2b37d432194
